navit/android/AndroidManifest.xml

```diff
@@ -23,6 +23,7 @@
             android:label="@string/app_name"
             android:theme="@style/NavitTheme"
             android:launchMode="singleTask"
+            android:exported="false"
```

Comment:
I am getting this error: 12-07 08:25:20.621  3635  3635 E BaseDraggingActivity: java.lang.SecurityException: Permission Denial: starting Intent { act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=org.navitproject.navit/.Navit bnds=[296,242][549,537] mCallingUid=10319 } from ProcessRecord{8b63d1f 3635:com.android.launcher/u0a319} (pid=3635, uid=10319) not exported from uid 10721 

So this one needs to be exported.

Reply:
done